test-cases/performance/perf-regression-throughput-125gb-with-ebs-gp3-volume.yaml

```diff
@@ -0,0 +1,15 @@
+
+stress_cmd_w: "cassandra-stress write no-warmup cl=ALL n=125000000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=100 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=1..125000000"
```

Comment:
The cmd_w should be cl=QUORUM, only preapre should be ALL.

Reply:
fixed